README.md

```diff
@@ -48,6 +48,21 @@ Content:
    You should now have a working kubernetes cluster.
    You should also have an encrypted kubeconfig at `<CK8S_CONFIG_PATH>/.state/kube_config_<prefix>.yaml` that you can use to access the cluster.
 
+## Changing authorized SSH keys for a cluster
+
+Authorized SSH keys can be changed for a cluster using:
+
+```bash
+./bin/ck8s-kubespray apply-ssh <prefix> [<options>]
+```
+
+It will set the public SSH key(s) found in`<CK8S_CONFIG_PATH>/<prefix>-config/group_vars/all/ck8s-ssh-keys.yaml` as authorized keys in your cluster (just add the keys you want to be authorized as elements in `ck8s_ssh_pub_keys_list`).
+Note that the authorized SSH keys for the cluster will be set to these keys _exclusively_, removing any keys that may already be authorized, so make sure the list includes **every SSH key** that should be authorized.
+
+When running this command, the SSH keys are applied to each node in the cluster sequentially, first the master and then the workers.
```

Comment:
I think it would be pretty nice to run first on workers and then control-plane nodes. Otherwise, if something goes wrong you will lose a control-plane node. Worst case: you have a single control-plane node so you loose the whole cluster. :scream:

Reply:
Yeah, this seems smart. I changed it.